### ISSUE TITLE
Timeout errors break SDK

### ISSUE THREAD

**Issue by nemo** [facebook/facebook-nodejs-ads-sdk#19]
Hey folks,

If a request times out (happens on our end if we're hammering the API quickly), then the SDK crashes.

Here's the error:

```
{ handle: 14,
  type: 'error',
  className: 'Error',
  constructorFunction: { ref: 15 },
  protoObject: { ref: 16 },
  prototypeObject: { ref: 11 },
  properties: 
   [ { name: 'stack', attributes: 2, propertyType: 0, ref: 17 },
     { name: 'message', attributes: 2, propertyType: 0, ref: 18 },
     { name: 'code', propertyType: 0, ref: 19 },
     { name: 'errno', propertyType: 0, ref: 19 },
     { name: 'syscall', propertyType: 0, ref: 20 },
     { name: 'hostname', propertyType: 0, ref: 21 },
     { name: 'host', propertyType: 0, ref: 21 },
     { name: 'port', propertyType: 0, ref: 22 } ],
  text: 'Error: getaddrinfo ENOTFOUND graph.facebook.com graph.facebook.com:443' }
```

It happens here: `/.../node_modules/facebook-nodejs-ads-sdk/dist/cjs.js:340`

**Comment by supasate** [comment]
Thanks @nemo for reporting. I'll work on it.

**Comment by supasate** [comment]
Hi @nemo, I'm not sure that your error is same with I found in #20 or not (I didn't see TypeError: Cannot read property 'error_user_msg' of undefined in your error). It might be related. If it's the same, I'll merge the issue.

**Comment by eranbetzalel** [comment]
#24 will also fix TIMEOUT errors.

**Comment by nemo** [comment]
@supasate that should be it!

**Comment by supasate** [comment]
Merged with #20.
